src/Sulu/Bundle/ContactBundle/Resources/config/services.xml

```diff
@@ -5,7 +5,7 @@
         <parameter key="sulu_contact.admin.class">Sulu\Bundle\ContactBundle\Admin\ContactAdmin</parameter>
         <parameter key="sulu_contact.contact_content_navigation_provider.class">Sulu\Bundle\ContactBundle\Admin\ContactContentNavigationProvider</parameter>
         <parameter key="sulu_contact.account_content_navigation_provider.class">Sulu\Bundle\ContactBundle\Admin\AccountContentNavigationProvider</parameter>
-        <parameter key="sulu_contact.js_config.class">Sulu\Bundle\AdminBundle\Admin\JsConfig</parameter>
+        <parameter key="sulu_contact.js_config.class">Sulu\Bundle\ContactBundle\Admin\JsConfigAvatarCollection</parameter>
```

Comment:
I'd see that as a chance to get rid of the parameter, since it is not recommended in symfony 3.0 anymore.